spyrmsd/graph.py

```diff
@@ -34,9 +122,18 @@
     "adjacency_matrix_from_atomic_coordinates",
 ]
 
-import numpy as np
+if len(_available_backends) == 0:
+    raise ImportError(
+        "No valid backends found. Please ensure that either graph-tool or NetworkX are installed."
+    )
+else:
+    if _current_backend is None:
+        ## Set the backend to the first available (preferred) backend
+        _current_backend = set_backend(backend=_available_backends[0])
```

Comment:
```suggestion
        ## Set the backend to the first available (preferred) backend
        set_backend(backend=_available_backends[0])
```
After the changes I suggested, this needs to change too (otherwise `_current_backend` is initialized to `None` by default).

Reply:
good catch, thanks